torch/jit/_shape_functions.py

```diff
@@ -724,6 +727,14 @@ def conv2d(
     assert len(input) == 4
     return conv_output_size(input, weight, bias, stride, padding, dilation, groups)
 
+def conv_backwards(grad_output: List[int], input:List[int], weight:List[int], biases:Optional[List[int]]):
+    if biases is None:
+        bias_result = None
+    else:
+        biases_2: List[int] = biases
+        bias_result = _copy(biases_2)
+
+    return _copy(input), _copy(weight), bias_result
```

Comment:
@eellison Is there an easy way to warn users if they forget `_copy`?

Reply:
Well, that lint check is likely broken, as it didn't fire in my original version of the the schema.